.travis.yml

```diff
@@ -30,6 +30,10 @@ matrix:
           env: SYMFONY_VERSION='^2'
         - php: 7.1
           env: SYMFONY_VERSION='^3'
+        - php 7.1
```

Comment:
you need `- php: 7.1` (missing colon)
same with the next entry

Reply:
ok, one moments, thx

Reply:
tests passed